DMX.Core.Api/Controllers/LabsController.cs

```diff
@@ -85,5 +86,47 @@ public async ValueTask<ActionResult<List<Lab>>> GetAllLabsAsync()
                 return InternalServerError(labOrchestrationServiceException);
             }
         }
+
+        [HttpDelete("{labId}")]
+#if RELEASE
+        [RequiredScope(RequiredScopesConfigurationKey = "AzureAd:Scopes:DeleteLab")] 
```

Comment:
@hassanhabib Do I need to create another scope for DeleteLab and pass it through provisioning first?

Reply:
Hum. that's a good question. We can go one of two routes:
1. Read scope covers (GET and GET ALL)
2. Write scope covers (PUT, POST and DELETE)

or the other route which is a scope for each action. I think assigning the same scope for this as `POST` is okay.

@terrypalo what do you think?

Reply:
Yeah we'll need another scope for DeleteLab and pass it through provisioning first. Every time we add a new endpoint, we should add new configuration to secure that endpoint. We can reuse scopes for those endpoints, but to ensure that each endpoint is secure in the way we want we'll need to create separate configuration

Reply:
@mimendel any thoughts on Terry's direction?
cc: @AndrewMcClelland && @geetika-jain

Reply:
I think for our current use cases, using Read/Write scopes are sufficient. However, I can foresee situations where we may want more granular access control over who has access to each endpoint.

For example, I could imagine wanting to grant access to someone to be able to register a new lab to DMX, but we don't want them to have access to wipe out all of the lab records we have.
i.e. - Grant them POST or ADD access, but not DELETE

Reply:
Thats true. I feel like its better to add the configuration per endpoint right now, so that we can configure everything without having to make big (code) changes in the future. It will make sure that our controllers follow some standard

Reply:
Sounds good to me :)

Reply:
Added in #101 
Thanks for the discussion!